response.go

```diff
@@ -45,9 +47,33 @@ func checkResponse(resp *http.Response, headerCondition func(string) Issues) (he
 // To interpret `issues`, see the list of conventions in the
 // documentation for Issues.
 func PreloadableResponse(resp *http.Response) (header *string, issues Issues) {
-	return checkResponse(resp, PreloadableHeaderString)
+	return checkResponse(resp, PreloadableHeaderStringWrapper, preloadlist.Bulk1Year)
```

Comment:
`PreloadableHeaderStringWrapper` here can just be `EligibleHeaderString`, which will allow for removing PreloadableHeaderStringWrapper.

Another option is that PreloadableResponse just calls EligibleResponse.